app/views/events/appointment.html

```diff
@@ -6,7 +6,7 @@
 {% set showNavigation = true %}
 {% set activeTab = 'appointment' %}
 {% set formAction = "./start" %}
-{% set allowEdits = event.status != "event_cancelled" and event.status != "event_rescheduled" %}
+{% set allowEdits = event.status != "event_cancelled" and event.status != "event_rescheduled" and event.status != "event_complete" %}
```

Comment:
```suggestion
{% set allowEdits = !(event | isFinal) %}
```

Can we swap this to using a status filter?